src/modules/Builder/pages/BuilderApplet/BuilderApplet.utils.tsx

```diff
@@ -579,6 +579,7 @@ export const getNewPerformanceTask = ({
     [PerfTaskType.Touch]: getGyroscopeOrTouchItems(GyroscopeOrTouch.Touch),
     [PerfTaskType.ABTrailsMobile]: getABTrailsItems(DeviceType.Mobile),
     [PerfTaskType.ABTrailsTablet]: getABTrailsItems(DeviceType.Tablet),
+    [PerfTaskType.Unity]: getABTrailsItems(DeviceType.Mobile),
```

Comment:
This is essentially adding a `ABTrails` task with a different name. This task should try to implement its own task properties, which will fail to get added until the BE is updated.
![CleanShot 2024-08-05 at 14 42 54](https://github.com/user-attachments/assets/fe3eb262-90d7-420f-9c6f-0fff8ab8be7f)

Reply:
Because this is behind a Feature Flag I'd prefer to have the full implementation in this ticket, similar to how `ParagraphItem` was handled, meaning you can't save Applets which include a `Unity` task